app/views/users/show.html.erb

```diff
@@ -1,2 +1,48 @@
-<h1><%= @user.name %></h1>
-<!-- Other user details -->
+<!-- users/show.html.erb -->
+<!DOCTYPE html>
+<html>
+<head>
+  <title><%= @user.name %></title>
+  <link rel="stylesheet" href="/assets/users.css">
+</head>
+<body>
+  <div class="user-container">
+    <div class="user-image">
+      <% if @user.photo.present? %>
+        <%= image_tag @user.photo, alt: "Profile picture" %>
+      <% else %>
+        <p>No photo available</p>
+      <% end %>
+    </div>
+    <div class="user-details">
+      <h1><%= @user.name %></h1>
+      <div class="user-posts-counter">
+        <p>Number of posts: <%= @user.posts.count %></p>
+      </div>
+    </div>
+  </div>
+
+  <div class="user-bio-container">
+    <p><%= @user.bio %></p>
+  </div>
+
+  <% @user.posts.limit(3).each_with_index do |post, index| %>
+    <div class="post-container">
+      <h2>Post #<%= index + 1 %> by <%= post.author.name %></h2>
+      <p><%= post.content %></p>
+      <p>Comments: <%= post.comments.count %>, Likes: <%= post.likes.count %></p>
+
+      <h3>Recent Comments:</h3>
+      <ul class="comments-list">
+        <% post.recent_comments.each do |comment| %>
+          <li><span>Username: <%= comment.author.name %></span>, Comment: <%= comment.text %></li> <!-- Change 'comment.content' to 'comment.text' -->
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="see-all-posts-container">
+    <%= link_to "See All Posts", all_posts_user_path(@user), class: "see-all-posts-button" %>

```

Comment:
- [ ] Like indicated in the previous review, I cannot see view a users bio (http://127.0.0.1:3000/users/1)

![image](https://github.com/TayyabKM/blog-app/assets/12745474/669308f3-ac2f-4786-9325-1ac2dcec1bf6)

When I click on a particular user:

![image](https://github.com/TayyabKM/blog-app/assets/12745474/bedee3ea-3db8-497c-8465-47fab4c784f3)

Let's ensure to fix this issue.

Reply:
<img width="926" alt="image" src="https://github.com/TayyabKM/blog-app/assets/109720981/7fb24de9-e894-4386-b386-28ad3e11d27e">
when i click on a particular user: 
<img width="956" alt="image" src="https://github.com/TayyabKM/blog-app/assets/109720981/ee821d34-dc1e-4efb-9e6e-851b6e21498c">
@rloterh

Reply:
Hi @TayyabKM 

I'm attaching below the database entries I made for your project. I believe it's clearly identify from the database query for the first_user has an id of 1, hence if **http://127.0.0.1:3000/users/** correctly yields all the users like I indicated above, then **http://127.0.0.1:3000/users/1** should show just the first_user (as seen in the database query below)

**First user has an id of 1:**
![image](https://github.com/TayyabKM/blog-app/assets/12745474/eecadce3-6eb1-4162-ba27-74aaf1d6d441)